app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -175,32 +197,53 @@ function AcuantCapture(
     }
   }
 
+  /**
+   * @param {AcuantSuccessResponse} nextCapture
+   */
+  function onAcuantImageCaptureSuccess(nextCapture) {
+    const { image, cardType, dpi, glare, sharpness } = nextCapture;
+    const isAssessedAsGlare = glare < ACCEPTABLE_GLARE_SCORE;
+    const isAssessedAsBlurry = sharpness < ACCEPTABLE_SHARPNESS_SCORE;
+    const { width, height, data } = image;
+
+    let result;
+    if (isAssessedAsGlare) {
+      setOwnErrorMessage(t('errors.doc_auth.photo_glare'));
+      result = 'glare';
+    } else if (isAssessedAsBlurry) {
+      setOwnErrorMessage(t('errors.doc_auth.photo_blurry'));
+      result = 'blurry';
+    } else {
+      onChangeAndResetError(data);
+      result = 'success';
+    }
+
+    addPageAction({
+      key: 'documentCapture.acuantWebSDKResult',
+      label: 'IdV: Acuant web SDK photo analyzed',
+      payload: {
+        fieldName: name,
+        documentType: getDocumentTypeLabel(cardType),
+        width,
+        height,
+        dpi,
+        glare,
+        sharpness,
+        isAssessedAsGlare,
+        isAssessedAsBlurry,
```

Comment:
should we also include ACCEPTABLE_GLARE_SCORE as a key so if we ever change it, we can see if things were compared to the old or new value? the boolean value is useful for a quick reference/filtering

Reply:
> should we also include ACCEPTABLE_GLARE_SCORE as a key so if we ever change it, we can see if things were compared to the old or new value? the boolean value is useful for a quick reference/filtering

Hm, that sounds like it might be wise. The idea of having the `isAssessedAs*` separate from the scores themselves was to anticipate this future possibility. But it seems it'd be easier to track what that specific threshold is over time by just including it here in the payload.

I'll update 👍

Reply:
> should we also include ACCEPTABLE_GLARE_SCORE as a key so if we ever change it, we can see if things were compared to the old or new value? the boolean value is useful for a quick reference/filtering

Included `glareScoreThreshold` and `sharpnessScoreThreshold` as payload values in 4fae2b0.